### PR TITLE
feat(integrations): add clientId and redirectURI to connect_config for zapier

### PR DESCRIPTION
**Pull request by dannylwe**
- add clientId to connect config for zapier
- add redirectURI to connect config for zapier

- [ext-327](https://linear.app/nango/issue/EXT-327/update-zapier-integration-to-have-redirecturi-and-clientid-import-from)

### CODE REVIEW

File: packages/shared/providers.yaml [hunk 1]
```diff
@@ -6960,6 +6960,17 @@ zapier:
     refresh_params:
         grant_type: refresh_token
     docs: https://docs.nango.dev/integrations/all/zapier
+    connection_config:
+        clientId:
+            type: string
+            title: Client ID
+            description: Zapier client ID for zap
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml:6967]: Not sure I understand the description

File: packages/shared/providers.yaml [hunk 2]
```diff
@@ -6960,6 +6960,17 @@ zapier:
...
+        redirectURI:
+            type: string
+            title: Redirect URI
+            description: The redirect URI for your Zapier applicaition
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml:6971]:
```suggestion
            description: The redirect URI for your Zapier application
```

File: packages/shared/providers.yaml [hunk 3]
```diff
@@ -6960,6 +6960,17 @@ zapier:
...
+            pattern: '^[a-zA-Z0-9:/-]+$'
+            example: https://www.nango.dev
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml:6973]: if it's an uri it should be like this:
```
            format: uri
            pattern: '^https?://.*$'
```

**Comment by khaliqgant** [comment]
Closing this as the intended usage isn't what we want for our customer.

**Pull request closed without merging**